tests/accesscontrol/tests/access_control_namespace.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
 )
 
-var _ = Describe("Access-control namespace, ", func() {
+var _ = Describe("Access-control namespace", Serial, func() {
```

Comment:
Describe strings that end with a comma need to keep the comma: we use these test case names to match Polarion records (e.g. https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-51860 ) so they need to have the exact same characters

Reply:
I sort of figured there was a reason there was random commas at the end of those strings.

Reply:
Okay these are all resolved now.